### PR TITLE
add function bitmap max & min doc

### PR DESCRIPTION
**Pull request by blackstar-baba**
add function bitmap max & min doc

### CODE REVIEW

**Comment by blackstar-baba** [comment]
PTAL @stdpain

**Review by stdpain — DISMISSED** [review]

**Comment by stdpain** [comment]
LGTM @Dshadowzh @wangsimo0

**Comment by wangsimo0** [comment]
@hellolilyliuyi Please check.

File: sql-reference/sql-functions/bitmap-functions/bitmap_min.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## description
+
+### Syntax
+
+```Haskell
+BITMAP BITMAP_MIN(BITMAP bitmap)
+```
+
+Get the minimum value in bitmap. if bitmap is null, return null. if bitmap is empty, return -1 by default.
```
Comment (jaogoy) [inline sql-reference/sql-functions/bitmap-functions/bitmap_min.md:11]: why return -1 by default, while BITMAP_MAX will return 0 by default if bitmap is empty.

Comment (blackstar-baba) [reply sql-reference/sql-functions/bitmap-functions/bitmap_min.md:11]: Look this PR: https://github.com/StarRocks/starrocks/pull/3293, We follow the rule of Postgresql bitmap after  discuss with @stdpain  & @Dshadowzh

**Comment by hellolilyliuyi** [comment]
Thank you for your contribution!!!

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by evelynzhaojie** [comment]
两个函数rd均已重新提pr，已发布官网。

**Pull request closed without merging**